index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+    <main>
+      <div class="container">
+        <div class="name">
+          <ul class="na">
+            <li>Emma Ngufor</li>
+          </ul>
+        </div>
+  
+        <div class="name1">
+          <ul class="name2">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><img src="image/Union.png" alt="an envelope"></li>
+          </ul>
+        </div>
+      </div>
+  
+      <h1>Hey There.<br> I'm Emmanuella</h1>
+      <h4> I'am a Software Developer</h4>
+      <p>I can help you build a product, feature or a website look through some 
+        of my work and experiment if you like what you see and have a project you need 
+        coded, don't hestiate to contact me. 
+      </p>
+  
+      <div class="content-image">
+        <img src="image/Illustration-header@2x.png" alt="a beautiful design">
+      </div>
```

Comment:
It's better to use the background-image property for placing the image 🤗

- _If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info._